Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -906,6 +915,8 @@ public override void Tick()
                     }
                 }
             }
+            if (shotHeight < 2.0f && originInt.DistanceTo(positionInt) > 3)
```

Comment:
reference to CollisionVertical.WallCollisionHeight would be good

Reply:
will do